src/Appwrite/Utopia/Response/Model/UsageUsers.php

```diff
@@ -82,14 +71,14 @@ public function __construct()
      */
     public function getName(): string
     {
-        return 'UsageUsers';
+        return static::class;
```

Comment:
Unfortunately, I think this will also return the namespace, which we don't want.